muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/webpack.config.js

```diff
@@ -81,7 +81,8 @@ module.exports = {
 	entry: entries,
 	output: {
 		filename: "[name].js",
-		path: __dirname + "/../dist"
+		path: __dirname + "/../dist",
+		jsonpFunction: "webpackjsonpFunction3"
```

Comment:
Where did this come from?....
Does not seem to affect anything nevertheless since there's no jsonp usage.
Maybe we should consider revert?...

Reply:
Seems to be coming from this commit: https://github.com/otavanopisto/muikku/commit/bff6985452453019fa6a36a4e070cf7234eaf4b7

It was needed when we compiled our front end with 3rd party library (we needed to use webpack3 then) and it's not relevant anymore as we have gotten rid of converseJs.

Should be reverted in my opinion.

Reply:
Glad someone knew what it was :D All for revert as well